cfme/infrastructure/cluster.py

```diff
@@ -117,7 +117,12 @@ class All(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self):
-        self.parent_view.navigation.select('Compute', 'Infrastructure', 'Clusters')
+        try:
```

Comment:
Nope. It depends on what type of provider did we connect. For all providers it is 'Clusters', and only for Openstack - 'Deployment Roles'